collectors/scm/gitlab/src/main/java/com/capitalone/dashboard/collector/GitlabGitCollectorTask.java

```diff
@@ -1,15 +1,21 @@
 package com.capitalone.dashboard.collector;
 
 import com.capitalone.dashboard.gitlab.DefaultGitlabGitClient;
+import com.capitalone.dashboard.gitlab.GitlabGitClient;
+import com.capitalone.dashboard.misc.HygieiaException;
 import com.capitalone.dashboard.model.Collector;
 import com.capitalone.dashboard.model.CollectorItem;
 import com.capitalone.dashboard.model.CollectorType;
 import com.capitalone.dashboard.model.Commit;
+import com.capitalone.dashboard.model.CommitType;
+import com.capitalone.dashboard.model.GitRequest;
 import com.capitalone.dashboard.model.GitlabGitRepo;
 import com.capitalone.dashboard.repository.BaseCollectorRepository;
 import com.capitalone.dashboard.repository.CommitRepository;
 import com.capitalone.dashboard.repository.ComponentRepository;
+import com.capitalone.dashboard.repository.GitRequestRepository;
 import com.capitalone.dashboard.repository.GitlabGitCollectorRepository;
+import org.apache.commons.collections.CollectionUtils;
```

Comment:
if possible please use the latest version of CollectionUtils

Reply:
Fixed by: 85964659d3324e7d043cf00c159e4ba662754a1e